README.md

```diff
@@ -344,7 +344,7 @@ A number of options are available to allow you to customise the SDK:
   The custom options are:
   - documentTypes (object)
 
-  The list of document types visible to the user can be filtered by using the `documentTypes` option. The default value for each document type is `true`.
+  The list of document types visible to the user can be filtered by using the `documentTypes` option. The default value for each document type is `true`. If only one document type is selected, the user will not see the document selection screen, instead they will see the capture screen straight away.
```

Comment:
Not sure, but if we replace

```
If only one document type is selected, the....
```

to:

```
If only one document type is set, the....
```

Isn't it clearer?

Reply:
I've changed it to ```If `documentTypes` only includes one document type, users will not see the document selection screen, instead they will be taken to the capture screen straight away.```